README.md

```diff
@@ -46,12 +44,21 @@ newman.run(
         reporters: "@reportportal/reportportal",
         reporter: {
             "@reportportal/reportportal": {
-                debug: true,
                 endpoint: "http://your-instance.com:8080/api/v1",
                 token: "00000000-0000-0000-0000-000000000000",
                 launch: "LAUNCH_NAME",
                 project: "PROJECT_NAME",
-                description: "LAUNCH_DESCRIPTION"
+                description: "LAUNCH_DESCRIPTION",
+                attributes: [
+                    {
+                        "key": "launchKey",
+                        "value": "launchValue"
+                    },
+                    {
+                        "value": "launchValue"
+                    },
+                ],
+                debug: true
```

Comment:
What about `mode` property for the launch destination?

Reply:
Thank you, added this functionality